Source/EasyNetQ.Management.Client/Serialization/StringObjectReadOnlyDictionaryConverter.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.ObjectModel;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace EasyNetQ.Management.Client.Serialization;
+
+internal sealed class StringObjectReadOnlyDictionaryConverter : JsonConverter<IReadOnlyDictionary<string, object?>>
+{
+    public override IReadOnlyDictionary<string, object?> Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        var jsonElement = JsonElement.ParseValue(ref reader);
+
+        switch (jsonElement)
+        {
+            case { ValueKind: JsonValueKind.Array }:
+                return new ReadOnlyDictionary<string, object?>(new Dictionary<string, object?>());
```

Comment:
Why do you return empty dictionary without reading array elements?

Reply:
In practice, it may not be empty?

Reply:
It should not.

Reply:
😄  should not != may not

Reply:
Feel free to investigate it in sources of management plugin.